src/Core/Validation/VerbValidation.fs

```diff
@@ -26,6 +26,10 @@ let hasRequiredInfoImperative =
     tryGetVerb
     >> hasImperative
 
+let hasRequiredInfoConjugation = 
```

Comment:
Note this is the same check as for the participles. You can opt to refactor code a little although I am okay with this as well.

Reply:
this was done intentionally, I would prefer to keep those separate :-)

Reply:
Okay, totally fine